Actions/AL-Go-Helper.ps1

```diff
@@ -1952,3 +1952,44 @@ function GetProject {
     }
     $project
 }
+
+function Get-NavSipFromArtifacts() {
+    #TODO: It would be nice with a different approach here - This downloads a lot of unnecessary stuff
+    $artifactTempFolder = Join-Path $([System.IO.Path]::GetTempPath()) ([System.IO.Path]::GetRandomFileName())
+    $navSipTempFolder = Join-Path $([System.IO.Path]::GetTempPath()) ([System.IO.Path]::GetRandomFileName())
+
+    try {
+        Download-Artifacts -artifactUrl (Get-BCArtifactUrl -type Sandbox) -includePlatform -basePath $artifactTempFolder | Out-Null
+        Write-Host "Downloaded artifacts to $artifactTempFolder"
+        $navsip = Get-ChildItem -Path $artifactTempFolder -Filter "navsip.dll" -Recurse
+        Write-Host "Found navsip at $($navsip.FullName)"
+        New-Item -Path $navSipTempFolder -ItemType Directory -Force -Verbose | Out-Null
+        Copy-Item -Path $navsip.FullName -Destination "$navSipTempFolder/navsip.dll" -Force | Out-Null
+        Write-Host "Copied navsip to $navSipTempFolder"
+    } finally {
+        Remove-Item -Path $artifactTempFolder -Recurse -Force
+    }
+    
+    return Join-Path $navSipTempFolder "navsip.dll" -Resolve
+}
+
+function Register-NavSip() {
+    $navsipPath = Get-NavSipFromArtifacts
+    $navSip32Path = "C:\Windows\System32"
+    $navSip64Path = "C:\Windows\SysWow64"
+
+    $navSip32Path, $navSip64Path | ForEach-Object {
```

Comment:
I don't think the SysWow64 is needed here - I don't think we don't ship the NavSip in 32 bit anymore